src/server/bridges/LivechatBridge.ts

```diff
@@ -133,6 +134,12 @@ export abstract class LivechatBridge extends BaseBridge {
        }
     }
 
+    public async do_fetchLivechatRoomMessages(appId: string, roomId: string): Promise<Array<IMessage>> {
+        if (this.hasMultiplePermission(appId)) {
```

Comment:
The `hasMultiplePermission` method checks for an existing `livechat-department.multiple` permission granted to the app, but the correct permission for this method is `livechat-message.multiple`

Reply:
Done! I did copy the wrong function